README.md

```diff
@@ -106,6 +106,8 @@ http://swagger.mapotempo.com/?url=http://optimizer.mapotempo.com/0.1/swagger_doc
 curl -X POST --header "Content-Type:application/json" --data '{"vrp":{"vehicles":[]}}' http://localhost:1791/0.1/vrp/submit.json?api_key=demo
 ```
 
+Moreover, the concepts used are described in the [docs](https://github.com/braktar/optimizer-api/blob/migrate-wiki/docs/Home.md) folder.
```

Comment:
You used your own link :) Would something like this work here `[docs](docs/Home.md)` or does it have to be complete link?